web/src/settings_config.ts

```diff
@@ -513,6 +513,48 @@ export const expires_in_values = {
     },
 };
 
+export const realm_deactivation_in_values = {
+    immediately: {
+        value: 0,
+        description: $t({defaultMessage: "Immediately"}),
+        default: false,
+    },
+    fourteenDays: {
```

Comment:
I think we can use snake_case variables here instead of camelCase variables. I think we can update the ones in `expires_in_values` as well, since all others seem to use snake_case.